pkg/clustermanager/wireguard.go

```diff
@@ -48,14 +48,14 @@ Endpoint = %s:51820
 }
 
 // PrivateIPPrefix extracts the first 3 digits of an IPv4 address
```

Comment:
/IPv4 address/CIDR Block

Reply:
@JohnnyQQQQ thx to take a look to this PR!
IMHO is not correct, we really extract the first 3 digits of an IPv4 address, I added that we extract if from CIDR block, let me know if it's ok.

Reply:
Yeah, you're right 😃